fs/indexed-pkg-importer/src/importIndexedDir.ts

```diff
@@ -28,6 +29,14 @@ export async function importIndexedDir (
       await moveOrMergeModulesDirs(path.join(newDir, 'node_modules'), path.join(stage, 'node_modules'))
     }
     await renameOverwrite(stage, newDir)
+    if (opts.makeExportableManifest) {
+      const manifest = await readProjectManifest(newDir)
+      const exportableManifest = await createExportableManifest(
+        path.dirname(filenames[manifest.fileName]),
+        manifest.manifest
+      )
+      await manifest.writeProjectManifest(exportableManifest)
+    }
```

Comment:
This is going to be used only by the deploy command, so I don't see any reason to put this code to importIndexedDir. Just place it in `deploy.ts`

Reply:
Updated to put all `createExportableManifest` logic in `deploy.ts`.

Removed all changes to other packages (adding `createExportableManifest` option)